src/App.vue

```diff
@@ -825,10 +825,7 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
         const error = new Error('Missing Payment Token or Filing ID')
         this.$root.$emit('save-error-event', error)
       }
-      this.setIsFilingPaying(false)
```

Comment:
Upon loading a page isFilingPaying, isSaving and isSavingResuming are false so we don't need to set them to false for situations where we leave the page.

Reply:
I'm thinking you should add your comment to the code, so that it doesn't look like someone forgot to add the line.

Also in Actions.vue.

Reply:
Will do!

Reply:
Will do